matrix-vis.js

```diff
@@ -105,11 +105,20 @@ function initGraph() {
     network = new vis.Network(container, data, options);
     // add event listeners
     network.on('select', function(params) {
-        var text = "";
         if (params.nodes.length === 1) {
-            text = JSON.stringify(nodes.get(params.nodes[0]).blob, undefined, 2);
+            t1="Event Type : "+nodes.get(params.nodes[0]).blob.type;
```

Comment:
`t1 t2 t3` are all being declared as global variables, please put `var` on them to scope them to the function.